orttraining/orttraining/python/training/ortmodule/configuration/__init__.py

```diff
@@ -0,0 +1,6 @@
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# __init__.py
+
+from .debug_options import DebugOptions, LogLevel, LoggingOptions, SaveOnnxOptions
```

Comment:
Public namespace proposal should be in the design doc for discussion as it affects user experience

Why do we need a new namespace for configuration?
This creates the single entry point for configuration similar to the master Configuration class that was eliminated from previous design iterations.

The encompassing namespace should be complete and allow users to fully control their API. Using additional namespace makes the code more verbose/complex

Reply:
After discussion, only exposing ```DebugOptions``` and ```LogLevel``` to user.